dev/bots/test.dart

```diff
@@ -153,6 +153,11 @@ Future<Null> _runTests() async {
   await _runSmokeTests();
 
   await _runFlutterTest(path.join(flutterRoot, 'packages', 'flutter'));
+  // Only packages/flutter/test/widgets/widget_inspector_test.dart really
```

Comment:
Is there a performance impact to this? If no, It should be ok to run all tests with that flag just to get more coverage.

Reply:
Running the tests with the flag takes only slightly longer than running the tests without the flag.

Reply:
OK, let's se what the impact is and if it gets too slow we can always revisit.